src/main/resources/application.yaml

```diff
@@ -518,6 +518,10 @@ notification:
     updateOtherPartyData:
       other_party:
         docmosisId: TB-SCS-GNO-ENG-00784.docx
+    sORExtendTime:
```

Comment:
Can we put this and other entries in an alphabetical order please.

Reply:
This section isnt already in alphabetical order (but cover sheets was so ordered that), I don't want to reorder this or otherwise it'll clunk up the PR with mainly refactoring